fmn/rules/taskotron.py

```diff
@@ -29,8 +46,15 @@ def taskotron_task(config, message, task=None):
     if not task:
         return False
 
+    name = message['msg']['task'].get('name').lower()
     tasks = [item.strip().lower() for item in task.split(',')]
-    return message['msg']['task'].get('name').lower() in tasks
+    tasks = [item for item in tasks if item]
```

Comment:
Maybe drop this line and just do ``if task and fnmatch.fnmatchcase(name, task):``